gmgc_mapper_plotter/GMGC_Mapper_plotter.ipynb

```diff
@@ -0,0 +1,222 @@
+      "text/plain": [
+       "<Figure size 720x720 with 1 Axes>"
+      ]
+     },
+     "metadata": {
+      "needs_background": "light"
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "hist_2= df[1].value_counts()\n",
+    "k_2 = len(df[1])\n",
+    "graf_3 = hist_2.plot(kind='barh', figsize=(10, 10), color='powderblue', zorder=5, width=0.5)\n",
+    "graf_3.tick_params(axis=\"both\", which=\"both\", bottom=\"off\", top=\"off\", labelbottom=\"on\", left=\"off\", right=\"off\", labelleft=\"on\")\n",
+    "vals_3 = graf_2.get_xticks()\n",
+    "for tick in vals_3:\n",
+    "      graf_3.axvline(x=tick, linestyle='dashed', alpha=0.8, color='lightgrey', zorder=1)\n",
```

Comment:
This is inconsistent (5 spaces here, 4 spaces below; 4 spaces is more standard).